src/games/strategy/triplea/ai/proAI/util/ProUtils.java

```diff
@@ -46,6 +47,26 @@ public ProUtils(final ProAI ai) {
     return unitTerritoryMap;
   }
 
+  public List<PlayerID> getEnemyPlayersInTurnOrder(final PlayerID player) {
```

Comment:
This class has examples of mockito: https://github.com/triplea-game/triplea/blob/a1554a51f4033105dd500759a14e72c6783263fd/test/games/strategy/engine/framework/startup/mc/GameSelectorModelTest.java